enterprise/suppress-user/internal/badgerdb/badgerdb.go

```diff
@@ -20,6 +20,8 @@ import (
 	"github.com/rudderlabs/rudder-server/utils/misc"
 )
 
+var ErrKeyNotFound = errors.New("key not found")
```

Comment:
We can move this to  `enterprise/suppress-user/model/model.go` and use it in both badger and memory implementation